### PR TITLE
Added memtester for testing faulty memory subsystem

### PR DESCRIPTION
**Pull request by harish-24**
Memtester test for is a utility for testing the memory subsystem in a computer to determine if it is faulty.

Signed-off-by: Harish <harish@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
this is V1 from #83

**Comment by PraveenPenguin** [comment]
@harish-24 job log please

**Comment by harish-24** [comment]
[job_memtestster.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/773746/job_memtestster.txt)

File: memory/memtester.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        for pkg in ['gcc', 'make']:
+            if not smm.check_installed(pkg) and not smm.install(pkg):
+                self.skip('%s is needed for the test to be run' % pkg)
+        tarball = self.fetch_asset(
+            'https://github.com/jnavila/memtester/archive/master.zip',
+            expire='7d')
+        archive.extract(tarball, self.srcdir)
+        self.srcdir = os.path.join(self.srcdir, 'memtester-master')
+        os.chdir(self.srcdir)
+        process.system('chmod 755 extra-libs.sh', shell=True, sudo=True)
```
Comment (PraveenPenguin) [inline memory/memtester.py]: please use ignore_status=True process.system problem with running command script will error out

**Comment by harish-24** [comment]
Closing this. Raised new PR #274

**Pull request closed without merging**